lib/identity/index.js

```diff
@@ -237,6 +237,51 @@ export default class Identity extends CryptoConsumer {
     )
     /* eslint-enable */
 
+    const brokerToznyOTPKeyNoteName = await this.crypto.genericHash(
```

Comment:
I don't mind long methods when all related to one action, but for readability and maintainability I think it would be helpful here to add some comments about what this particular block of code among all the other block of code is doing.